datasets/sentinel-3/collection/sentinel-3-synergy-aod-l2-netcdf/template.json

```diff
@@ -0,0 +1,235 @@
+                "partition_frequency": "AS"
+            }
+        }
+    },
+    "summaries": {
+        "constellation": [
+            "Sentinel-3"
+        ],
+        "platform": [
+            "Sentinel-3A",
+            "Sentinel-3B"
+        ],
+        "instruments": [
+            "OLCI",
+            "SLSTR"
+        ],
+        "sat:platform_international_designator": [
+            "2016-011A",
+            "2018-039A"
+        ],
+        "sat:orbit_state": [
+            "ascending",
+            "descending"
+        ],
+        "eo:cloud_cover": {
+            "minimum": 0,
+            "maximum": 100
+        },
+        "s3:product_type": [
+            "SY_2_AOD___"
+        ],
+        "s3:product_name": [
+            "AOD"
+        ],
+        "s3:processing_timeliness": [
+            "NT"
+        ],
+        "s3:saline_water": {
+            "minimum": 0,
+            "maximum": 100
+        },
+        "s3:land": {
+            "minimum": 0,
+            "maximum": 100
+        },
+        "eo:bands": [
+            {
+                "name": "SYN_440",
+                "description": "OLCI channel Oa03",
+                "center_wavelength": 0.4425,
+                "full_width_half_max": 0.01
+            },
+            {
+                "name": "SYN_550",
+                "description": "SLSTR nadir and oblique channel S1",
+                "center_wavelength": 0.55,
+                "full_width_half_max": 0.02
+            },
+            {
+                "name": "SYN_670",
+                "description": "SLSTR nadir and oblique channel S2",
+                "center_wavelength": 0.659,
+                "full_width_half_max": 0.02
+            },
+            {
+                "name": "SYN_865",
+                "description": "OLCI channel Oa17, SLSTR nadir and oblique channel S2",
+                "center_wavelength": 0.865,
+                "full_width_half_max": 0.02
+            },
+            {
+                "name": "SYN_1600",
+                "description": "SLSTR nadir and oblique channel S5",
+                "center_wavelength": 1.61,
+                "full_width_half_max": 0.06
+            },
+            {
+                "name": "SYN_2250",
+                "description": "SLSTR nadir and oblique channel S6",
+                "center_wavelength": 2.25,
+                "full_width_half_max": 0.05
+            }
+        ]
+    },
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -180,
+                    -90,
+                    180,
+                    90
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2020-04-16T19:36:28.012367Z",
+                    null
+                ]
+            ]
+        }
+    },
+    "item_assets": {
+        "safe-manifest": {
+            "description": "SAFE product manifest",
```

Comment:
In the items, are we leaving these as `description`, or moving them to `title`?

Reply:
I've been using descriptions for everything. Many are pithy enough to serve as a title, but many read more like a description. So I've defaulted to using a description for everything. I don't want to use both a title and description, in which case we'll likely end up with descriptions that almost match the title.